pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -55,6 +55,14 @@ const (
 	Skip
 )
 
+const (
+	// MaxNodeScore is the maximum score a Score plugin is expected to return.
+	MaxNodeScore int = 100
```

Comment:
this should be equal to MaxPriority.

Reply:
done

Reply:
I meant literally equal, as in: ```MaxNodeScore = schedulerapi.MaxPriority```

are there dependency issues that prevent us from importing scheduler api here?

Reply:
Could you explain a little bit why we should use the const instead of `10` here? I think there is a chance that the two different const could be different, I don't like to establish such a connection between the two const unless you have some strong statements.

Reply:
I don't think they should be different, they are semantically the same, MaxPriority will be deprecated once we re-write all priority functions as score plugins. Why do you they could be different?

Reply:
> I don't think they should be different, they are semantically the same,

They are semantically the same (to some extent), but they are used in different parts of the scheduler, one if used in the priority functions and extender, another one is used in the scheduling framework. I think it would be better if we treat them separately.

> MaxPriority will be deprecated once we re-write all priority functions as score plugins. Why do you they could be different?

Are you suggesting we are going to expand the range after we rewrite all the priority function? And before that, we have to set `MaxNodeScore` to 10 for compatibility?

Reply:
It's confusing to me to keep two things that has the same purpose. I agree with @ahg that we should try to consolidate them into one single thing.

Reply:
> > I don't think they should be different, they are semantically the same,
> 
> They are semantically the same (to some extent), but they are used in different parts of the scheduler, one is used in the priority functions and extender, another one is used in the scheduling framework. I think it would be better if we treat them separately since they belong to different packages and modules.

Score Plugins are just another form of Priority functions, and so **ideally** MaxPriority and NodeMaxScore should be the same to avoid confusion and potential bugs that could be caused by them being set to different values without properly re-adjusting the scores when ranking the nodes at the end.

> 
> > MaxPriority will be deprecated once we re-write all priority functions as score plugins. Why do you they could be different?
> 
> Are you suggesting we are going to expand the range after we rewrite all the priority function? And before that, we have to set `MaxNodeScore` to 10 for compatibility?

No, we will try to do it sooner, this will depend on the discussion in the issue you created. The ideal end solution is to set ```MaxNodeScore = MaxPriority ``` and ```MaxPriority = 100```. Whether or not we end up with this ideal solution, or back to what we had initially in this PR (i.e., use a multiplier and set them to different values) depends on the answer to the question "why MaxPriority is exposed in the scheduler API".

Reply:
> "why MaxPriority is exposed in the scheduler API".

IMO MaxPriority has already been exposed in the scheduler API, and are you suggesting now we need to find out the reason of why we did this before?

Reply:
Yes, so that we understand how to change its value, and how to deprecate it eventually.

Reply:
> Yes, so that we understand how to change its value, and how to deprecate it eventually.

Addressed, please take another look.